redash/models/users.py

```diff
@@ -172,49 +172,49 @@ def profile_image_url(self):
     @property
     def permissions(self):
         # TODO: this should be cached.
-        return list(itertools.chain(*[g.permissions for g in Group.query.filter(Group.id.in_(self.group_ids))]))
+        groups = db.session.scalars(select(Group).where(Group.id.in_(self.group_ids))).all()
+        return list(itertools.chain(*[g.permissions for g in groups]))
 
     @classmethod
     def get_by_org(cls, org):
-        return cls.query.filter(cls.org == org)
+        return select(cls).where(cls.org == org)
```

Comment:
it requires to change almost every method names of all model classes

Reply:
I don't think there are many methods that return partial queries rather than objects or collections of objects, but what do you think?

I think this update updates all classes related to SQLAlchemy, is this correct? If so, I think it would be smart to make the above changes in this update. (If I create this, I would like to try that). This is because there is rarely an opportunity to update all classes related to SQLAlchemy.

However, even before this update, the naming of the method that returns a partial query is ambiguous, it is fine to change such method names (optional).